extensions/wikia/DesignSystem/DesignSystemHelper.class.php

```diff
@@ -60,6 +60,21 @@ public static function getSvg( $name, $classNames = '', $alt = '' ) {
 		}
 	}
 
+	public static function getApiImage( $model, $classNames = '', $alt = '' ) {
+		if ( $model['type'] === 'wds-svg' ) {
+			return static::getSvg( $model['name'], $classNames, $alt );
+		} elseif ($model['type'] === 'image-external') {
+			return Html::element( 'img', [
```

Comment:
This class is already tightly coupled to MW. We already use `wfMessage()` and `WikiaLogger` there. Do you think it makes sense to try avoiding libraries that we already integrated with MW?